ts/components/conversation/message/message-content/MessageContextMenu.tsx

```diff
@@ -330,14 +336,12 @@ export const MessageContextMenu = (props: Props) => {
               <Item onClick={onDeleteForEveryone}>{unsendMessageText}</Item>
             </>
           ) : null}
-          {weAreAdmin && isPublic ? <Item onClick={onBan}>{window.i18n('banUser')}</Item> : null}
-          {weAreAdmin && isPublic ? (
-            <Item onClick={onUnban}>{window.i18n('unbanUser')}</Item>
-          ) : null}
-          {weAreAdmin && isPublic && !isSenderAdmin ? (
+          {showAdminActions ? <Item onClick={onBan}>{window.i18n('banUser')}</Item> : null}
```

Comment:
Small thing but can't we group these together? Like this:

```jsx
{showAdminActions ? 
  <>
    <Item onClick={onBan}>{window.i18n('banUser')}</Item>
    <Item onClick={onUnban}>{window.i18n('unbanUser')}</Item>
  </>
: null}
```

Reply:
ok, I merged everything together

```ts
          {showAdminActions ? (
            <>
              <Item onClick={onBan}>{window.i18n('banUser')}</Item>
              <Item onClick={onUnban}>{window.i18n('unbanUser')}</Item>
              {isSenderAdmin ? (
                <Item onClick={removeModerator}>{window.i18n('removeFromModerators')}</Item>
              ) : (
                <Item onClick={addModerator}>{window.i18n('addAsModerator')}</Item>
              )}
            </>
          ) : null}